ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts

```diff
@@ -156,6 +157,9 @@ export class TestPlanPlatformOsVersionFormComponent implements OnInit {
         if (!this.platformBrowser || setValue) {
           this.platformBrowser = this.browsers[0];
           this.environmentFormGroup?.controls['browser']?.setValue(this.platformBrowser?.id);
+          if(this.version.workspace.isMobileWeb && this.platformBrowser.isChrome){
+            this.environmentFormGroup?.controls['browser']?.setValue(Browsers.CHROME);
```

Comment:
plz configure prettier to avoid unnecessary spacing

Reply:
Done